solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Home/HomeView.swift

```diff
@@ -43,27 +51,50 @@ final class HomeView: UIView {
         self.restaurants = restaurants
         self.tableView.reloadData()
     }
+    
+    func updateLoading(with isLoading: Bool) {
+        if isLoading {
+            loadingView.isHidden = !isLoading
+            loadingView.loadingSpinner.startAnimating()
+            loadingView.setLoadingMessage("Carregando Restaurantes...")
+        } else {
+            loadingView.isHidden = !isLoading
+            loadingView.loadingSpinner.stopAnimating()
+        }
+    }
```

Comment:
O `loadingView.isHidden = !isLoading` pode ficar fora do if.
```suggestion
    func updateLoading(with isLoading: Bool) {
        loadingView.isHidden = !isLoading
        
        if isLoading {
            loadingView.loadingSpinner.startAnimating()
            loadingView.setLoadingMessage("Carregando Restaurantes...")
        } else {
            loadingView.loadingSpinner.stopAnimating()
        }
    }
```